easybuild/easyconfigs/d/Doxygen/Doxygen-1.8.10-GNU-4.9.3-2.25.eb

```diff
@@ -0,0 +1,26 @@
+easyblock = 'CMakeMake'
```

Comment:
@boegel Yes, byt that works with ConfigureMake (with a non-standard prefix, therefore needed the easyblock), but from 1.8.10 only CMake build, and it seems to work fine (meanwhile the easybock not)